stories/tooltip.js

```diff
@@ -151,14 +159,6 @@ storiesOf('Tooltip', module)
         I'm small text with a <TooltippedBadge tooltip={textSmallTooltipContent}>hover me</TooltippedBadge>{' '}
         tooltip action
       </TextSmall>
-
-      <TextTiny marginTop={4}>
-        I'm small text with a{' '}
-        <TooltippedBadge tooltip={textSmallTooltipContent}>
-          hover me
-        </TooltippedBadge>{' '}
-        tooltip action
-      </TextTiny>
```

Comment:
Q: This used to be TextTiny and it got changed to TextDisplay? Shouldn't it have been changed to TextSmall?